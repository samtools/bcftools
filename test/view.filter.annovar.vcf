##fileformat=VCFv4.0
##fileDate=20090805
##source=myImputationProgramV3.1
##reference=1000GenomesPilot-NCBI36
##phasing=partial
##INFO=<ID=NS,Number=1,Type=Integer,Description="Number of Samples With Data">
##INFO=<ID=DP,Number=1,Type=Integer,Description="Total Depth">
##INFO=<ID=AF,Number=.,Type=Float,Description="Allele Frequency">
##INFO=<ID=AA,Number=1,Type=String,Description="Ancestral Allele">
##INFO=<ID=DB,Number=0,Type=Flag,Description="dbSNP membership, build 129">
##INFO=<ID=H2,Number=0,Type=Flag,Description="HapMap2 membership">
##FILTER=<ID=q10,Description="Quality below 10">
##FILTER=<ID=s50,Description="Less than 50% of samples have data">
##FORMAT=<ID=GT,Number=1,Type=String,Description="Genotype">
##FORMAT=<ID=GQ,Number=1,Type=Integer,Description="Genotype Quality">
##FORMAT=<ID=DP,Number=1,Type=Integer,Description="Read Depth">
##FORMAT=<ID=HQ,Number=2,Type=Integer,Description="Haplotype Quality">
##INFO=<ID=ANNOVAR_DATE,Number=1,Type=String,Description="Flag the start of ANNOVAR annotation for one alternative allele">
##INFO=<ID=Gene.refGene,Number=.,Type=String,Description="Gene.refGene annotation provided by ANNOVAR">
##INFO=<ID=GeneDetail.refGene,Number=.,Type=String,Description="GeneDetail.refGene annotation provided by ANNOVAR">
##INFO=<ID=ExonicFunc.refGene,Number=.,Type=String,Description="ExonicFunc.refGene annotation provided by ANNOVAR">
##INFO=<ID=AAChange.refGene,Number=.,Type=String,Description="AAChange.refGene annotation provided by ANNOVAR">
##INFO=<ID=Func.ensGene,Number=.,Type=String,Description="Func.ensGene annotation provided by ANNOVAR">
##INFO=<ID=Gene.ensGene,Number=.,Type=String,Description="Gene.ensGene annotation provided by ANNOVAR">
##INFO=<ID=GeneDetail.ensGene,Number=.,Type=String,Description="GeneDetail.ensGene annotation provided by ANNOVAR">
##INFO=<ID=ExonicFunc.ensGene,Number=.,Type=String,Description="ExonicFunc.ensGene annotation provided by ANNOVAR">
##INFO=<ID=AAChange.ensGene,Number=.,Type=String,Description="AAChange.ensGene annotation provided by ANNOVAR">
##INFO=<ID=esp6500si_ea,Number=1,Type=Float,Description="esp6500si_ea annotation provided by ANNOVAR">
##INFO=<ID=esp6500si_all,Number=1,Type=Float,Description="esp6500si_all annotation provided by ANNOVAR">
##INFO=<ID=1000g2012apr_eur,Number=1,Type=Float,Description="1000g2012apr_eur annotation provided by ANNOVAR">
##INFO=<ID=1000g2012apr_all,Number=1,Type=Float,Description="1000g2012apr_all annotation provided by ANNOVAR">
##INFO=<ID=snp138,Number=.,Type=String,Description="snp138 annotation provided by ANNOVAR">
##INFO=<ID=LJB2_SIFT,Number=.,Type=String,Description="LJB2_SIFT annotation provided by ANNOVAR">
##INFO=<ID=LJB2_PolyPhen2_HDIV,Number=.,Type=String,Description="LJB2_PolyPhen2_HDIV annotation provided by ANNOVAR">
##INFO=<ID=LJB2_PP2_HDIV_Pred,Number=.,Type=String,Description="LJB2_PP2_HDIV_Pred annotation provided by ANNOVAR">
##INFO=<ID=LJB2_PolyPhen2_HVAR,Number=.,Type=String,Description="LJB2_PolyPhen2_HVAR annotation provided by ANNOVAR">
##INFO=<ID=LJB2_PolyPhen2_HVAR_Pred,Number=.,Type=String,Description="LJB2_PolyPhen2_HVAR_Pred annotation provided by ANNOVAR">
##INFO=<ID=LJB2_LRT,Number=.,Type=String,Description="LJB2_LRT annotation provided by ANNOVAR">
##INFO=<ID=LJB2_LRT_Pred,Number=.,Type=String,Description="LJB2_LRT_Pred annotation provided by ANNOVAR">
##INFO=<ID=LJB2_MutationTaster,Number=1,Type=String,Description="LJB2_MutationTaster annotation provided by ANNOVAR">
##INFO=<ID=LJB2_MutationTaster_Pred,Number=.,Type=String,Description="LJB2_MutationTaster_Pred annotation provided by ANNOVAR">
##INFO=<ID=LJB_MutationAssessor,Number=.,Type=String,Description="LJB_MutationAssessor annotation provided by ANNOVAR">
##INFO=<ID=LJB_MutationAssessor_Pred,Number=1,Type=String,Description="LJB_MutationAssessor_Pred annotation provided by ANNOVAR">
##INFO=<ID=LJB2_FATHMM,Number=.,Type=String,Description="LJB2_FATHMM annotation provided by ANNOVAR">
##INFO=<ID=LJB2_GERP++,Number=.,Type=String,Description="LJB2_GERP++ annotation provided by ANNOVAR">
##INFO=<ID=LJB2_PhyloP,Number=.,Type=String,Description="LJB2_PhyloP annotation provided by ANNOVAR">
##INFO=<ID=LJB2_SiPhy,Number=.,Type=String,Description="LJB2_SiPhy annotation provided by ANNOVAR">
##INFO=<ID=ALLELE_END,Number=0,Type=Flag,Description="Flag the end of ANNOVAR annotation for one alternative allele">
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	NA00001	NA00002	NA00003
16	50745926	rs2066844	C	T	80	PASS	NS=3;DP=14;AF=0.5;DB;H2;ANNOVAR_DATE=2015-06-17;Func.refGene=exonic;Gene.refGene=NOD2;GeneDetail.refGene=.;ExonicFunc.refGene=nonsynonymous_SNV;AAChange.refGene=NOD2:NM_001293557:exon3:c.C2023T:p.R675W,NOD2:NM_022162:exon4:c.C2104T:p.R702W;Func.ensGene=exonic;Gene.ensGene=ENSG00000167207;GeneDetail.ensGene=.;ExonicFunc.ensGene=nonsynonymous_SNV;AAChange.ensGene=ENSG00000167207:ENST00000300589:exon4:c.C2104T:p.R702W;esp6500si_ea=0.043488;esp6500si_all=0.031558;1000g2012apr_eur=0.05;1000g2012apr_all=0.02;snp138=rs2066844;LJB2_SIFT=0.010000;LJB2_PolyPhen2_HDIV=0.999;LJB2_PP2_HDIV_Pred=D;LJB2_PolyPhen2_HVAR=0.901;LJB2_PolyPhen2_HVAR_Pred=P;LJB2_LRT=0.993490;LJB2_LRT_Pred=N;LJB2_MutationTaster=0.291000;LJB2_MutationTaster_Pred=N;LJB_MutationAssessor=2.32;LJB_MutationAssessor_Pred=medium;LJB2_FATHMM=-0.5;LJB2_GERP++=3.66;LJB2_PhyloP=1.421000;LJB2_SiPhy=6.9139;ALLELE_END	GT:GQ:DP:HQ	0|0:48:1:51,51	1|0:48:8:51,51	1/1:43:5:.,.
20	1230237	.	T	G	47	PASS	NS=3;DP=13;AA=T;ANNOVAR_DATE=2015-06-17;Func.refGene=intronic;Gene.refGene=RAD21L1;GeneDetail.refGene=.;ExonicFunc.refGene=.;AAChange.refGene=.;Func.ensGene=intronic;Gene.ensGene=ENSG00000244588;GeneDetail.ensGene=.;ExonicFunc.ensGene=.;AAChange.ensGene=.;esp6500si_ea=.;esp6500si_all=.;1000g2012apr_eur=.;1000g2012apr_all=.;snp138=.;LJB2_SIFT=.;LJB2_PolyPhen2_HDIV=.;LJB2_PP2_HDIV_Pred=.;LJB2_PolyPhen2_HVAR=.;LJB2_PolyPhen2_HVAR_Pred=.;LJB2_LRT=.;LJB2_LRT_Pred=.;LJB2_MutationTaster=.;LJB2_MutationTaster_Pred=.;LJB_MutationAssessor=.;LJB_MutationAssessor_Pred=.;LJB2_FATHMM=.;LJB2_GERP++=.;LJB2_PhyloP=.;LJB2_SiPhy=.;ALLELE_END	GT:GQ:DP:HQ	0|0:54:7:56,60	0|0:48:4:51,51	0/0:61:2
20	1230288	.	T	.	50	PASS	NS=3;DP=13;AA=T;ANNOVAR_DATE=2015-06-17;Func.refGene=intronic;Gene.refGene=RAD21L1;GeneDetail.refGene=.;ExonicFunc.refGene=.;AAChange.refGene=.;Func.ensGene=intronic;Gene.ensGene=ENSG00000244588;GeneDetail.ensGene=.;ExonicFunc.ensGene=.;AAChange.ensGene=.;esp6500si_ea=.;esp6500si_all=.;1000g2012apr_eur=.;1000g2012apr_all=.;snp138=.;LJB2_SIFT=.;LJB2_PolyPhen2_HDIV=.;LJB2_PP2_HDIV_Pred=.;LJB2_PolyPhen2_HVAR=.;LJB2_PolyPhen2_HVAR_Pred=.;LJB2_LRT=.;LJB2_LRT_Pred=.;LJB2_MutationTaster=.;LJB2_MutationTaster_Pred=.;LJB_MutationAssessor=.;LJB_MutationAssessor_Pred=.;LJB2_FATHMM=.;LJB2_GERP++=.;LJB2_PhyloP=.;LJB2_SiPhy=.;ALLELE_END	GT:GQ:DP:HQ	0|0:54:7:56,60	0|0:48:4:51,51	0/0:61:2
