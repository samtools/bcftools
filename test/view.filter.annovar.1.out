20	1230237	.	T	G	47	PASS	NS=3;DP=13;AA=T;ANNOVAR_DATE=2015-06-17;Func.refGene=intronic;Gene.refGene=RAD21L1;GeneDetail.refGene=.;ExonicFunc.refGene=.;AAChange.refGene=.;Func.ensGene=intronic;Gene.ensGene=ENSG00000244588;GeneDetail.ensGene=.;ExonicFunc.ensGene=.;AAChange.ensGene=.;esp6500si_ea=.;esp6500si_all=.;1000g2012apr_eur=.;1000g2012apr_all=.;snp138=.;LJB2_SIFT=.;LJB2_PolyPhen2_HDIV=.;LJB2_PP2_HDIV_Pred=.;LJB2_PolyPhen2_HVAR=.;LJB2_PolyPhen2_HVAR_Pred=.;LJB2_LRT=.;LJB2_LRT_Pred=.;LJB2_MutationTaster=.;LJB2_MutationTaster_Pred=.;LJB_MutationAssessor=.;LJB_MutationAssessor_Pred=.;LJB2_FATHMM=.;LJB2_GERP++=.;LJB2_PhyloP=.;LJB2_SiPhy=.;ALLELE_END	GT:GQ:DP:HQ	0|0:54:7:56,60	0|0:48:4:51,51	0/0:61:2:.
20	1230288	.	T	.	50	PASS	NS=3;DP=13;AA=T;ANNOVAR_DATE=2015-06-17;Func.refGene=intronic;Gene.refGene=RAD21L1;GeneDetail.refGene=.;ExonicFunc.refGene=.;AAChange.refGene=.;Func.ensGene=intronic;Gene.ensGene=ENSG00000244588;GeneDetail.ensGene=.;ExonicFunc.ensGene=.;AAChange.ensGene=.;esp6500si_ea=.;esp6500si_all=.;1000g2012apr_eur=.;1000g2012apr_all=.;snp138=.;LJB2_SIFT=.;LJB2_PolyPhen2_HDIV=.;LJB2_PP2_HDIV_Pred=.;LJB2_PolyPhen2_HVAR=.;LJB2_PolyPhen2_HVAR_Pred=.;LJB2_LRT=.;LJB2_LRT_Pred=.;LJB2_MutationTaster=.;LJB2_MutationTaster_Pred=.;LJB_MutationAssessor=.;LJB_MutationAssessor_Pred=.;LJB2_FATHMM=.;LJB2_GERP++=.;LJB2_PhyloP=.;LJB2_SiPhy=.;ALLELE_END	GT:GQ:DP:HQ	0|0:54:7:56,60	0|0:48:4:51,51	0/0:61:2:.
